
# Imp Syp iCLIP source data

This repo contains source data for the following publication:  

[![Publisher Link](https://github.com/jefflee1103/Imp-Syp-iCLIP-shiny-app/blob/main/app/www/science_advances_logo.png)](https://www.science.org/doi/10.1126/sciadv.adr6682)
> **Imp/IGF2BP and Syp/SYNCRIP temporal RNA interactomes uncover combinatorial networks of regulators of Drosophila brain development** Lee JY, et al. (2025) *Science Advances* > DOI: [10.1126/sciadv.adr6682](https://www.science.org/doi/10.1126/sciadv.adr6682)


## Please see live Shiny application for user-friendly data exploration

### Imp/Syp iCLIP Data Explorer

The application provides a user-friendly interface to browse the complete iCLIP dataset, search for specific genes, and visualize the RNA-binding protein (RBP) binding sites on the UCSC Genome Browser.

**The interactive web application can be accessed here:**

[**>> Launch Imp/Syp iCLIP Data Explorer <<**](https://jefflee1103.github.io/Imp-Syp-iCLIP-shiny-app/)

## Abstract
Temporal patterning of neural progenitors is an evolutionarily conserved mechanism generating neural diversity. In Drosophila, postembryonic neurogenesis requires the RNA binding proteins (RBPs) Imp/IGF2BP and Syp/SYNCRIP. However, how they coachieve their function is not well understood. Here, we elucidate the in vivo temporal RNA interactome landscapes of Imp and Syp during larval brain development. Imp and Syp bind a highly overlapping set of conserved mRNAs encoding proteins involved in neurodevelopment. We identify transcripts differentially occupied by Imp/Syp over time, featuring a network of known and previously unknown candidate temporal regulators that are post-transcriptionally regulated by Imp/Syp. Furthermore, the physical and coevolutionary relationships between Imp and Syp binding sites reveal a combinatorial, rather than competitive, mode of molecular interplay. Our study establishes an in vivo framework for dissecting the temporal coregulation of RBP networks as well as providing a resource for understanding neural fate specification.

Source data (including track bigWig/Bed files) used for the publication is available from this repo.  

Raw and processed FASTQ files are available from the Gene Expression Omnibus (GEO) [GSE262499](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE262499).

## Contact  
- jeff.lee{at}glasgow.ac.uk  
- ilan.davis{at}glasgow.ac.uk
- [Ilan Davis lab website](ilandavis.com)



